app/ui/src/main/res/drawable/ic_status_corner.xml

```diff
@@ -1,6 +1,8 @@
-<vector android:autoMirrored="true" android:height="24dp"
+<vector xmlns:tools="http://schemas.android.com/tools"
+    android:autoMirrored="true" android:height="24dp"
     android:viewportHeight="100.0" android:viewportWidth="49.999996"
-    android:width="12dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    android:width="12dp" xmlns:android="http://schemas.android.com/apk/res/android"
+    tools:targetApi="kitkat">
```

Comment:
Replace the `targetApi` attribute with `tools:ignore="UnusedAttribute,VectorRaster"`.